BL602 pull request

jczhang777 opened the pull request:
#### Problem
* Add new platform for matter

#### Change overview
Add chip bl602 for matter

#### Testing
How was this tested? (at least one bullet point required)
* see "connectedhomeip/examples/lighting-app/bouffalolab/bl602/README.md"

andy31415 commented:
@jczhang777 is this the same as #16358 ?

andy31415 reviewed (changes requested):
There seems to be some reviews lost in history as the same pull request is being created over and over again: https://github.com/project-chip/connectedhomeip/pulls?q=is%3Apr+bl602

See comments in #16283 - need build_examples support.

Also for history and review please keep the same PR and do not open new PRs for every change as that is very spammy

jczhang777 commented:
> There seems to be some reviews lost in history as the same pull request is being created over and over again: https://github.com/project-chip/connectedhomeip/pulls?q=is%3Apr+bl602
> 
> See comments in #16283 - need build_examples support.
> 
> Also for history and review please keep the same PR and do not open new PRs for every change as that is very spammy

sorry about that.

andy31415 commented:
> > There seems to be some reviews lost in history as the same pull request is being created over and over again: https://github.com/project-chip/connectedhomeip/pulls?q=is%3Apr+bl602
> > See comments in #16283 - need build_examples support.
> > Also for history and review please keep the same PR and do not open new PRs for every change as that is very spammy
> 
> sorry about that. i had fixed issues that #16283 mentioned。but i do not know why spell check failed in README.md. build_examples support is changed as you comments.

for build_examples.py to work I expect changes in `scripts/build/build/targets.py` as well as adding a new platform  and unit test updates. I do not see these in this PR.

andy31415 reviewed (commented):
Please remove README.pdf

git does not like binary files, we have a readme.md

andy31415 commented on examples/lighting-app/bouffalolab/bl602/README.md:
````diff
@@ -0,0 +1,81 @@
+*   Build the example application:
+
+```
+connectedhomeip$ ./scripts/examples/gn_bl602_example.sh ./examples/lighting-app/bouffalolab/bl602 out/lighting_app_bl602
````
I expect this to be like:

```
./scripts/build/build_examples.py --target bouffalolab-bl602-lighting-app build
```

jczhang777 commented:
> Restyler complains just about one empty line:
> 
> [380f3e1](https://github.com/project-chip/connectedhomeip/commit/380f3e1129de71d06af61816738292194c01fde1)

very thankful.

jczhang777 commented:
> Please remove README.pdf
> 
> git does not like binary files, we have a readme.md

ok.

jczhang777 commented:
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

this project is on going.

woody-apple commented:
/rebase

woody-apple reviewed (approved):
Fast tracking platform changes.

jczhang777 commented:
> Fast tracking platform changes.



> /rebase

working on it. thanks.

> Fast tracking platform changes.

ok, I am working on it.

jczhang777 commented:
@andy31415 Hi, I had rebased my code, and fix some problems.Include https://github.com/project-chip/connectedhomeip/pull/16370#discussion_r829411785 that you had mentioned。

jczhang777 commented:
Hi, I had rebased my code, and fix some problems.Include https://github.com/project-chip/connectedhomeip/pull/16370#discussion_r829411785 that you had mentioned。Are there any other issues that I need to fix? very thankful。

jczhang777 commented:
@andy31415 Hi, I had rebased my code, and fix some problems.Include https://github.com/project-chip/connectedhomeip/pull/16370#discussion_r829411785 that you had mentioned。Are there any other issues that I need to fix? very thankful。

bzbarsky-apple commented:
> Are there any other issues that I need to fix

The restyle failure.  See the automatically created commits in https://github.com/project-chip/connectedhomeip/pull/16921 for the needed changes.

jczhang777 commented:
> > Are there any other issues that I need to fix
> 
> The restyle failure. See the automatically created commits in #16921 for the needed changes.



> > Are there any other issues that I need to fix
> 
> The restyle failure. See the automatically created commits in #16921 for the needed changes.
I just fixed it. If there is anything else I need to modify, please let me know. thank you very much. 


> > Are there any other issues that I need to fix
> 
> The restyle failure. See the automatically created commits in #16921 for the needed changes.
I just fixed it. If there is anything else I need to modify, please let me know. thank you very much.

jczhang777 commented:
> > Are there any other issues that I need to fix
> 
> The restyle failure. See the automatically created commits in #16921 for the needed changes.

I just fixed it. If there is anything else I need to modify, please let me know. thank you very much.

bzbarsky-apple commented:
/rebase

bzbarsky-apple commented:
For some reason CI did not kick off properly there.. You might need to force-push again.  :(

jczhang777 commented:
> For some reason CI did not kick off properly there.. You might need to force-push again. :(

I have pushed forcibly pushed up. Please check if there is anything else to modify. Thank you.
